packages/select/src/Select.js

```diff
@@ -449,19 +473,27 @@ Select.propTypes = {
   feedback: PropTypes.bool,
   placeholder: PropTypes.string,
   components: PropTypes.object,
+  required: PropTypes.bool,
+  clearButtonClassName: PropTypes.string,
 };
 
 components.Option.propTypes = {
   innerProps: PropTypes.object,
   isSelected: PropTypes.bool,
 };
 
-components.ClearIndicator.propTypes = {
-  innerProps: PropTypes.object,
+components.Input.propTypes = {
+  selectProps,
 };
 
 components.MultiValueRemove.propTypes = {
   innerProps: PropTypes.object,
 };
 
+SelectStyles.propTypes = {
```

Comment:
If we make `SelectStyles` lowercase, then I don't think we need to specify propTypes here or in the `.d.ts` file

Reply:
The `prop-types` package is for components. When `SelectStyles` is uppercase the linter probably thinks it is a component instead of a regular function. If we make it lowercase then the linter won't throw an error. We should still have the types in `.d.ts`. I jumped the gun on that one